terraform/aws/modules/rds-aurora/main.tf

```diff
@@ -0,0 +1,101 @@
+resource "aws_rds_cluster" "this" {
+  cluster_identifier              = "${local.stack}-${var.rds_suffix}"
+  engine                          = var.db_engine_type
+  engine_version                  = var.db_engine_version
+  engine_mode                     = var.db_engine_mode
+  database_name                   = var.database_name
+  master_username                 = var.master_username
+  master_password                 = random_password.master_password.result
```

Comment:
do you see any terraform examples where secrets manager is used to store the master password? maybe this is something we can do in a future module version. there are lots of benefits to using secrets manager for RDS - here's some background info, you don't have to address now if you don't have time:

https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/rds-secrets-manager.html